core/services/vrf/vrf_request_cost_test.go

```diff
@@ -20,6 +20,7 @@ func TestMeasureRandomnessRequestGasCost(t *testing.T) {
 
 	assert.Greater(t, estimate, uint64(134000),
 		"requestRandomness tx gas cost lower than expected")
-	assert.Less(t, estimate, uint64(160000),
+	// Note: changed from 160000 to 164079 in the Berlin hard fork (Geth 1.10)
```

Comment:
cc @coventry I'm assuming we've done some pre-Berlin fork inspection of our contracts to ensure they won't be broken by these gas cost changes?